crates/tn-reth/src/lib.rs

```diff
@@ -734,6 +760,188 @@ impl RethEnv {
         Ok(sealed_block_with_senders)
     }
 
+    #[inline]
```

Comment:
This is harmless so can ignore for the PR but why all these inlines?  These function seem large to be inlined and also seem to be doing a lot so probably would be lost in the noise anyway.  It is unlikely you will outguess the optimizer or that it will even respect these.  Just seems like clutter to me but I may be missing something.